GiwazipClient/Views/PostingTextViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  PostingTextViewController.swift
+//  GiwazipClient
+//
+//  Created by creohwan on 2023/01/31.
+//
+
+import UIKit
+
+import SnapKit
+
+class PostingTextViewController: BaseViewController {
+    
+    // MARK: - Property
+    
+    private let textViewPlaceHolder: String = """
+                                            예시)
+                                            - 화장실이 빨강색이면 좋겠어요~!
+                                            - 집이 커졌으면 좋겠어요~!
+                                            """
+    
+    // MARK: - View
+    
+    private let textGuideLabel: UILabel = {
+        $0.text = "문의 내역을 작성해주세요"
+        $0.numberOfLines = 0
+        $0.textColor = .black
+        $0.font = UIFont.systemFont(ofSize: 18, weight: .semibold)
+        return $0
+    }(UILabel())
+    
+    private lazy var textView : UITextView = {
+        $0.text = textViewPlaceHolder
+        $0.textColor = .white
+        $0.backgroundColor = .lightGray
+        $0.font = UIFont.systemFont(ofSize: 18, weight: .regular)
+        $0.textContainerInset = UIEdgeInsets(top: 14, left: 14, bottom: 14, right: 14)
+        $0.textAlignment = .natural
+        $0.layer.cornerRadius = 10
+        $0.delegate = self
+        return $0
+    }(UITextView())
+    
+    private lazy var finishButton: UIButton = {
+        let buttonConfiguration = UIButton.Configuration.filled()
+        $0.configuration = buttonConfiguration
+        $0.configuration?.title = "문의하기"
+        $0.configuration?.attributedTitle?.font = UIFont.systemFont(ofSize: 18, weight: .semibold)
+        $0.configuration?.attributedTitle?.foregroundColor = .white
+        $0.configuration?.background.backgroundColor = .gray
```

Comment:
p1;
해당 부분을 아래와 같이 바꾸면 일일이 색상 조정할 필요 없이, 버튼의 `isEnabled` 속성만 바꾸는 것으로 색을 같이 바꿀 수 있습니다.
```suggestion
        $0.configuration?.baseForegroundColor = .white
        $0.configuration?.baseBackgroundColor = .blue
```

이 코드가 적용된다면 아래의 `textViewDidChange` function 내부에서 일일이 색상을 바꾸지 않고, `isEnabled` 값만 true, false로 조절해도 되겠네요.

Reply:
> p1; 해당 부분을 아래와 같이 바꾸면 일일이 색상 조정할 필요 없이, 버튼의 `isEnabled` 속성만 바꾸는 것으로 색을 같이 바꿀 수 있습니다.
> 
> 이 코드가 적용된다면 아래의 `textViewDidChange` function 내부에서 일일이 색상을 바꾸지 않고, `isEnabled` 값만 true, false로 조절해도 되겠네요.

923a0ea 오호 이런 방법이~! base 친구들을 쓰면 아주 좋군요!

Reply:
> > p1; 해당 부분을 아래와 같이 바꾸면 일일이 색상 조정할 필요 없이, 버튼의 `isEnabled` 속성만 바꾸는 것으로 색을 같이 바꿀 수 있습니다.
> > 이 코드가 적용된다면 아래의 `textViewDidChange` function 내부에서 일일이 색상을 바꾸지 않고, `isEnabled` 값만 true, false로 조절해도 되겠네요.
> 
> [923a0ea](https://github.com/Samsamhada/Giwazip-Client-App/commit/923a0ea98215c8ef829936e678739aabfed5a34b) 오호 이런 방법이~! base 친구들을 쓰면 아주 좋군요!

확인했습니다.